README.md

```diff
@@ -3,31 +3,45 @@
 Code to calculate the FEXERJ rating based on chess-results pages. Code is still somewhat rough. Some functionality, documentation and unit/integration testing are missing.
 
 
-## How to use
+# How to use
 
-There are two CSV needed: one for players with established rating and another for players with temporary rating. The separator should be a semicolon ";". The first CSV should have the following header/columns:
+## Pre-conditions
 
-Id_No;Title;Name;Rtg_Nat;ClubName;Birthday;Sex;Fed;TotalNumGames
+There are two CSV needed: one for players and another for tournaments to be processed. The separator should be a semicolon ";". The first CSV should have the following header/columns:
 
-while the second CSV should have:
+`Id_No;Title;Name;Rtg_Nat;ClubName;Birthday;Sex;Fed;TotalNumGames;AvgOpponRating;TotalPoints`
 
-Id_No;Title;Name;Rtg_Nat;ClubName;Birthday;Sex;Fed;TotalNumGames;AvgOpponRating;TotalPoints
+While the second CSV should have:
 
+`#;CR_id;Name;EndDate;Type;isIRT?;isFEXERJ?`
+
+All players in the tournaments should already be in the first CSV.
+
+## How to run
 
 The user should run the following command:
 
-```python3 calculator.py```
+`python fexerj-rating-calculator.py <tournament_list_file> <first_torunament_to_run> <number_of_tournaments_to_run> <players_list_file>`
+
+The program will create one intermediate players' list file for each tournament in the following format:
+`RatingList_after_<number of the tournament>.csv`
+
+The program will also create a JSON file with all the players which ID was entered manually by the user. This is to avoid having to enter them again in a rerun. This file needs to be deleted by the user if the tournaments' file change.
```

Comment:
As a possible future improvement we could add a field to the json with the tournament file name and hash. This way we can automatically delete it if needed.

Could you file a feature request for this (and maybe other comments above):